specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/TranslatorBatch.json

```
@@ -117,22 +117,70 @@
           {
             "in": "query",
             "name": "$top",
-            "description": "Take the $top entries in the collection\r\nWhen both $top and $skip are supplied, $skip is applied first",
+            "description": "$top indicates the total number of records the user wants to be returned across all pages.\r\n            \r\nClients MAY use $top and $skip query parameters to specify a number of results to return and an offset into the collection.\r\nWhen both $top and $skip are given by a client, the server SHOULD first apply $skip and then $top on the collection.\r\n            \r\nNote: If the server can't honor $top and/or $skip, the server MUST return an error to the client informing about it instead of just ignoring the query options.",
             "type": "integer",
             "format": "int32",
-            "default": 50,
-            "maximum": 100,
-            "minimum": 1
+            "maximum": 2147483647,
+            "minimum": 0
           },
           {
             "in": "query",
             "name": "$skip",
-            "description": "Skip the $skip entries in the collection\r\nWhen both $top and $skip are supplied, $skip is applied first",
+            "description": "$skip indicates the number of records to skip from the list of records held by the server based on the sorting method specified.  By default, we sort by descending start time.\r\n            \r\nClients MAY use $top and $skip query parameters to specify a number of results to return and an offset into the collection.\r\nWhen both $top and $skip are given by a client, the server SHOULD first apply $skip and then $top on the collection.\r\n            \r\nNote: If the server can't honor $top and/or $skip, the server MUST return an error to the client informing about it instead of just ignoring the query options.",
             "type": "integer",
             "format": "int32",
             "default": 0,
             "maximum": 2147483647,
             "minimum": 0
+          },
+          {
+            "in": "query",
+            "name": "$maxpagesize",
+            "description": "$maxpagesize is the maximum items returned in the items in a page.  If more items are requested via $top (or $top is not specified and there are more items to be returned), @nextLink will contain the link to the next page.\r\n            \r\nClients MAY request server-driven paging with a specific page size by specifying a $maxpagesize preference. The server SHOULD honor this preference if the specified page size is smaller than the server's default page size.",
```

consider updating to 
```suggestion
            "description": "$maxpagesize is the maximum items returned in a page.  If more items are requested via $top (or $top is not specified and there are more items to be returned), @nextLink will contain the link to the next page.\r\n            \r\nClients MAY request server-driven paging with a specific page size by specifying a $maxpagesize preference. The server SHOULD honor this preference if the specified page size is smaller than the server's default page size.",
```

---

oops, didn't realize i can just commit your changes...i made this manually already.